spec/features/user_stories_spec.rb

```diff
@@ -0,0 +1,55 @@
+
+describe "user stories" do
+
+  # PERSONAS
+  # Kylie and Marc, ordering a take out meal
+
+  let(:menu) { Menu.new }
+  let(:order) { Order.new }
+
+  # As a customer
+  # So that I can check if I want to order something
+  # I would like to see a list of dishes with prices
+  it 'so that Marc can see if wants to order something, he should be able to see a list of dishes with prices' do
+    expect{ menu.view }.not_to raise_error
+  end
+
+  # As a customer
+  # So that I can order the meal I want
+  # I would like to be able to select some number of several available dishes
+  xit 'so that Kylie can order what she wants, she should be able to choose from available dishes' do
```

Comment:
Metrics/LineLength: Line is too long. [101/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)